examples/expression-oriented-style/expression_oriented_style.md

```diff
@@ -0,0 +1,8 @@
+Ballerina supports statements for familiarity but also tries to enable an expression-oriented style to be more intuitive.
+For example instead of using an if statement Ballerina allows, 
```

Comment:
```suggestion
For example, instead of using an if statement, Ballerina allows the below. 
```

Reply:
Incomplete sentence? 

May be we can bring the expression into the sentence and say

```suggestion
For example, instead of using an `if` statement, Ballerina allows `return x > 0 ? 1 : 0;`.
```